tools/cmake/toolchains/GCC_ARM.cmake

```diff
@@ -7,6 +7,14 @@ set(CMAKE_CXX_COMPILER "arm-none-eabi-g++")
 set(GCC_ELF2BIN "arm-none-eabi-objcopy")
```

Comment:
isn't this part of the toolchain file ? Someelse could use different executable?

Reply:
I am not sure what you mean.
`GCC_ARM.cmake` is only included if the Mbed toolchain file is included.

Reply:
I see, it is fine here. The other bit below confused me.